webapp/src/main/resources/public/js/components/workbench/TranslationHistoryModal.js

```diff
@@ -0,0 +1,122 @@
+import PropTypes from 'prop-types';
+import React from "react";
+import {FormattedMessage, injectIntl} from "react-intl";
+import {Button, Modal} from "react-bootstrap";
+import {Table} from "react-bootstrap";
+import StatusGlyph from '../widgets/StatusGlyph';
+import TextUnitSDK from "../../sdk/TextUnit";
+
+class translationHistoryModal extends React.Component {
+    static propTypes() {
+        return {
+            "show": PropTypes.bool.isRequired,
+            "textUnit": PropTypes.object.isRequired,
+            "translationHistory": PropTypes.object.isRequired
+        };
+    }
+
+    renderHistoryItem = (item) => {
+        return item ?
+            (
+                <tr>
+                    <td>{item.createdByUser.username}</td>
```

Comment:
vertical alignment (text - icon) seems off? maybe because of the icon size?

Reply:
After addressing Jee's comment below, it is aligned now